client/devdocs/design/gutenberg-blocks.jsx

```diff
@@ -0,0 +1,26 @@
+/**
+ * External dependencies
+ */
+import React from 'react';
+import classnames from 'classnames';
+
+/**
+ * Internal dependencies
+ */
+import DocumentHead from 'components/data/document-head';
+import Main from 'components/main';
+import ReadmeViewer from 'components/readme-viewer';
+
+
+export default class GutenbergBlocks extends React.Component {
+	render() {
+		const className = classnames( 'devdocs', 'devdocs__gutenberg-blocks' );
+
+		return (
+			<Main className={ className }>
+				<DocumentHead title="Gutenberg Blocks"/>
```

Comment:
Re: naming I think @mtias suggested "Gutenberg Components" for now, though I'm not sure how strongly he feels about that.

Reply:
Gutenberg have both [blocks](https://github.com/WordPress/gutenberg/tree/master/core-blocks) and [components](https://github.com/WordPress/gutenberg/tree/master/packages/components). It's my understanding that we want to show only the blocks according to the project introduction @ehg gave to me, as nothing was said about the components.

Having that in consideration, do we want to name the blocks from Gutenberg as "Gutenberg components" in Calypso? Wouldn't it be confusing?

Reply:
Blocks in Calypso (more than pure UI components / handle their own data needs) need to be renamed to something else, but I'll leave it up to the framework group to decide what else.

> Wouldn't it be confusing?

Until this is resolved, yes, every variation is a bit confusing. 😄